en/django_start_project/README.md

```diff
@@ -73,6 +73,8 @@ Let's ignore the other files for now as we won't change them. The only thing to
 
 Let's make some changes in `mysite/settings.py`. Open the file using the code editor you installed earlier.
 
+**Note**: Keep in mind that `settings.py` is a regular file, like any other. You can open it from inside the code editor, using the "open -> select file". Alternatively, you can open the file by navigating to the djangogirls folder on your desktop and right-clicking on it. Then, select your code editor from the list. Selecting the editor is important as you might have other programs installed that can open the file but will not let you edit it.
```

Comment:
Is "open -> select file" specific to a particular editor? In Atom at least, it is something different.